deploy-agent/deployd/BUILD

```diff
@@ -0,0 +1,44 @@
+load("@rules_python//python:defs.bzl", "py_test", "py_library")
+load("@python_deps//:requirements.bzl", "requirement")
+
+py_library(
```

Comment:
shall we add the py_test targets ?

Reply:
yeah planed to add test targets next.

Reply:
Added some test targets. For some reasons, test_agent test is not really stable while running with bazel.